core/src/main/scala/org/bitcoins/core/api/feeprovider/FeeRateApi.scala

```diff
@@ -6,6 +6,6 @@ import scala.concurrent.Future
 
 trait FeeRateApi {
 
-  def getFeeRate: Future[FeeUnit]
+  def getFeeRate(): Future[FeeUnit]
```

Comment:
i had to modify this because i couldn't figure out how to mock a no argument method correctly 🤷‍♂️ . I can revert this if there is a solution 

https://github.com/bitcoin-s/bitcoin-s/pull/3973/files#diff-b5f2265432eb3e3a5f714135702e23e99dc538645bbe2a94ef69a062b553c479R28